.travis.yml

```diff
@@ -4,6 +4,19 @@
 # for it to be on multiple physical lines, so long as you remember: - There
 # can't be any leading "-"s - All newlines will be removed, so use ";"s
 language: python
+
+# Run jobs on container-based infrastructure, can be overridden per job
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.cache/pip
```

Comment:
Is that right ? 

I have the impression from http://docs.travis-ci.com/user/workers/container-based-infrastructure/ that caching is available for public repositories using container-based infrastructure.